cmd/edit/cluster/cmd.go

```diff
@@ -160,6 +163,13 @@ func init() {
 			"to be added to the Hosted Control Plane's VPC Endpoint Service to enable additional "+
 			"VPC Endpoint connection requests to be automatically accepted.",
 	)
+
+	flags.StringVar(
+		&args.billingAccount,
+		"billing-account",
+		"",
+		"Account used for billing subscriptions purchased via the AWS marketplace",
```

Comment:
If this is about ROSA, customers don't directly purchase ROSA through the AWS marketplace. They do so through the AWS console as ROSA is offered as a 1st party service through AWS

Reply:
This could be a better wording "Account used for billing subscriptions purchased via the AWS console for ROSA"